frontend/util.lua

```diff
@@ -1014,21 +1015,22 @@ end
 -- Note that if PDF file size is around 1024, 4096, 16384, 65536, 262144
 -- 1048576, 4194304, 16777216, 67108864, 268435456 or 1073741824, appending data
 -- by highlighting in KOReader may change the digest value.
-function util.partialMD5(file)
-    local bit = require("bit")
-    local md5 = require("ffi/sha2").md5
-    local leftshift = bit.lshift
+function util.partialMD5(filepath)
+    if not filepath then return end
+    local file = io.open(filepath, 'rb')
+    if not file then return end
     local step, size = 1024, 1024
     local update = md5()
     for i = -1, 10 do
-        file:seek("set", leftshift(step, 2*i))
+        file:seek("set", lshift(step, 2*i))
```

Comment:
Was this a typo? But it was working? Are these 2 names available? Why the change? :)